services/db-cleaner/email-templates/email_template.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title>Title</title>
+  <link rel="stylesheet" href="foundation-emails.css" />
+</head>
+
+<style>
+  body,
+  html,
+  .body {
+    background: #f3f3f3 !important;
+  }
+
+  .top-section {
+    background: #26374A;
+  }
+
+  .sensor-bucket-logo {
+    padding: 1em;
+    width: 65%;
+  }
+
+  .centered {
+    text-align: center;
+  }
+</style>
+
+<body>
+  <!-- <style> -->
+  <table class="body" data-made-with-foundation="">
+    <tr>
+      <td class="float-center" align="center" valign="top">
+        <center data-parsed="">
+          <table align="center" class="container float-center">
+            <tbody>
+              <tr>
+                <td>
+                  <table class="row">
+                    <tbody>
+                      <tr>
+                        <th>
+                          <table>
+                            <tr>
+                              <th>
+                                <center class="top-section">
+                                  <center class="sensor-bucket-logo" data-parsed=""> <img
+                                      src="http://localhost:3000/static/logo-white.png" align="center"
+                                      class="float-center"> </center>
+                                </center>
+                              </th>
+                            </tr>
+                          </table>
+                        </th>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table class="row">
+                    <tbody>
+                      <tr>
+                        <th class="small-12 large-12 first last">
+                          <table>
+                            <tr>
+                              <th>
+                                <br>
+                                <p class="lead">Geplande Gegevensopruiming</p>
+                                <p>Via deze weg willen wij u informeren dat de maandelijkse gegevensopruiming staat gepland
+                                  op:</p>
+                                <br><br>
+                                <p class="centered"><strong>{{.DeletionDate}}</strong></p><br><br>
+                                <p>U heeft nog {{.DaysTillDeletion}} dagen om de data van uw apparaten veilig te stellen.</p>
+                                <p>Alleen de data van de afgelopen 30 dagen zal verwijderd worden.</p>
```

Comment:
De archiefperiode verschilt per organisatie en mogelijk per apparaat. Wellicht verwoorden naar iets als:
"Data ouder dan de ingestelde archiefperiode van de sensor of van de organisatie zal worden opgeschoond"
oid

Reply:
done